src/request/notification/options.rs

```diff
@@ -19,6 +19,54 @@ impl<'a> CollapseId<'a> {
     }
 }
 
+#[derive(Default, Debug, Clone, PartialEq, Eq)]
+/// The apns-push-type header field has the following valid values.
+/// The descriptions below describe when and how to use these values.
+/// Send an apns-push-type header with each push. Recent and upcoming features
+/// may not work if this header is missing. See the table above to determine if
+/// this header is required or optional.
+///
+/// see https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/sending_notification_requests_to_apns#4294485
+pub enum PushType {
```

Comment:
Looks like `complication` is variant missing, but this may have been added in a more recent version.